lib/src/test/java/grpcbridge/common/TestService.java

```diff
@@ -51,6 +53,16 @@ public void get(
         responseObserver.onCompleted();
     }
 
+    @Override
+    public void postWrappers(PostWrappersRequest request,
```

Comment:
Could you reformat to match other methods in the file?

Reply:
Reformatted.